jaas/src/main/java/com/griddynamics/jagger/jaas/rest/TestEnvironmentRestController.java

```diff
@@ -84,24 +108,33 @@ public TestEnvironmentRestController(TestEnvironmentService testEnvironmentServi
     }
 
     @PostMapping(consumes = APPLICATION_JSON_VALUE)
+    @ApiOperation(value = "Creates Test Environment.", notes = "If operation successful, 'Environment-Session' cookie is returned with response, "
+            + "which is required for PUT request. This cookie is valid only for Test Environment with envId which was specified in request body.")
+    @ApiResponses(value = {
+            @ApiResponse(code = 201, message = "Creation is successful."),
+            @ApiResponse(code = 400, message = "envId doesn't match " + ENV_ID_PATTERN + " OR status doesn't correspond to runningTestSuite value."),
+            @ApiResponse(code = 409, message = "Test Environment with provided envId already exists.")})
```

Comment:
'status doesn't correspond to runningTestSuite value' - not clear what is meant here